src/qibolab/platform.py

```diff
@@ -233,11 +218,11 @@ def update(self, updates: dict):
 
                 # classification
                 elif par == "mean_gnd_states":
-                    self.qubits[qubit].mean_gnd_states = str(value)
+                    self.qubits[qubit].mean_gnd_states = [float(voltage) for voltage in value]
```

Comment:
This pattern is so frequent (throughout both Qibolab and Qibocal) that we could define a `complist()` function.

Reply:
If it's just for this line, we could also do `list(map(float, value))`, but I think it is simple enough as it is. Wouldn't be more complicated to define a function here and import it in qibocal (in terms of maintenance, etc.)?

Reply:
Ok, maybe I've seen it much more in Qibocal.

What I had in mind is that this is the replacement for complex numbers, that are used in many places (if I haven't misunderstood the usage).
Since they should be treated consistently all over the place, having a single function would ensure this consistency.

In any case, it's not critical, it was just a minor annotation. We could also face the issue in a later refactor, if it will appear again.